pkg/assembler/backends/testing/backend.go

```diff
@@ -114,10 +114,6 @@ func GetBackend(args backends.BackendArgs) (backends.Backend, error) {
 	if err != nil {
 		return nil, err
 	}
-	err = registerAllHasSLSA(client)
```

Comment:
do we no longer want to add data to the testing backend?

Reply:
Does this need to be replaced by another? Or ingestion via the client only?

Reply:
Via client/playground is ok here.

Don't remove the others yet as we might use them for testing/perf